docraptor.gemspec

```diff
@@ -1,4 +1,26 @@
 # -*- encoding: utf-8 -*-
+#
+=begin
+#DocRaptor v1
+
+OpenAPI spec version: 1.1.0
+
+Generated by: https://github.com/swagger-api/swagger-codegen.git
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=end
+
```

Comment:
I find these level of commenting pretty annoying and unnecessary. I think some of our post-generation scripts may already be removing this. I would prefer we do that for Ruby as well. I have an open discussion with swagger-codegen on some alternatives to this banner stuff.